backend/src/main/java/com/daedan/festabook/global/exception/DuplicateDataException.java

```diff
@@ -0,0 +1,10 @@
+package com.daedan.festabook.global.exception;
+
+import org.springframework.http.HttpStatus;
+
+public class DuplicateDataException extends DatabaseException {
+
+    public DuplicateDataException(String originalExceptionMessage, HttpStatus status) {
+        super("중복된 데이터가 발생했습니다.", originalExceptionMessage, status);
+    }
+}
```

Comment:
```suggestion
@Getter
public class ConflictException extends BusinessException {

    private final Class<? extends BaseEntity> clazz;

    public ConflictException(Class<? extends BaseEntity> clazz) {
        this(String.format("%s 이미 존재합니다.", clazz.getSimpleName()), clazz);
    }

    protected ConflictException(String message, Class<? extends BaseEntity> clazz) {
        super(message, HttpStatus.CONFLICT);
        this.clazz = clazz;
    }
}
```

`ConflictException`과 비슷하게 Clazz를 받아서 사용하는건 어때요? 
모두 동일한 "중복된 데이터가 발생했습니다." 메시지를 응답한 이유가 있을까요?

Reply:
`.body("message", equalTo("FestivalNotification 이미 존재합니다."));`

이 형태로 나올 수 있게 비슷하게 구현하면 좋을 것 같긴합니다! 통일성으로